services/tables/src/main/java/com/linkedin/openhouse/tables/repository/impl/OpenHouseInternalRepositoryImpl.java.rej

```diff
@@ -0,0 +1,10 @@
+diff a/services/tables/src/main/java/com/linkedin/openhouse/tables/repository/impl/OpenHouseInternalRepositoryImpl.java b/services/tables/src/main/java/com/linkedin/openhouse/tables/repository/impl/OpenHouseInternalRepositoryImpl.java	(rejected hunks)
```

Comment:
Uploaded by mistake?

Reply:
Will fix this. Thanks for catching this.